syntax = "proto3";

option java_multiple_files = true;
option java_package = "com.staticanalyzer.algservice";
option java_outer_classname = "AlgServiceProto";

package algservice;

service AlgService {
   rpc JustReturn (JustReturnRequest) returns (JustReturnResponse) {}
};

message JustReturnRequest {
    bytes file = 1;
}

message JustReturnResponse {
    map<string, string> result = 1;
}